src/components/appointments/CreateAppointment/CreateAppointment.js

```diff
@@ -49,6 +49,8 @@ const CreateAppointment = () => {
   const reset = () => {
     clearSubmission();
     resetSession();
+    // clear any cached data or lingering submission state
+    window.sessionStorage.clear();
```

Comment:
Clearing the whole sessionStorage doesn't have any unwanted side-effects? This is per browser-tab, right?

Reply:
It is per browser tab indeed, and it should not have any unwanted side effects. The react-use hooks even deal with browser storage API's being blocked an in that case all state is purely in memory (using `useState`). So we should *never* assume that sessionStorage is guaranteed to be present.